### ISSUE TITLE
Velero Kopia does not respect excludedNamespace

### ISSUE THREAD

**Issue by slawekww** [vmware-tanzu/velero#5854]
**What steps did you take and what happened:**
Velero has been deployed on k8s cluster using helm chart and schedule has been defined to exclude namespaces from backup.
Once uploader=kopia is setup, excludedNamespaces is not respected - all namespaces are backup by kopia (persistent volumes).

In addition, I've observed that kopia runs backup for each node in sequence, it means wait to complete one node backup to start the next one even if node agents are run on each node in parallel.

**What did you expect to happen:**
Once uploader=kopia is setup, excludedNamespaces are respected.
Improve performance of kopia to run backup in parallel on many nodes. 

**The following information will help us better understand what's going on**:

Helm values to velero helm chart version 3.1.2:
```yaml
upgradeCRDs: false
image:
  tag: v1.10.1
labels:
  velero.io/exclude-from-backup: "true"
metrics:
  enabled: true
  scrapeTimeout: 30s
  scrapeInterval: 60s
  serviceMonitor:
    enabled: true
    additionalLabels:
      prometheus-servicemonitors: "true"
      velero.io/exclude-from-backup: "true"
backupsEnabled: true
snapshotsEnabled: true
deployNodeAgent: true
nodeAgent:
  privileged: true
  resources:
    limits:
      cpu: 2000m
      memory: 4096Mi
  nodeSelector:
    kubernetes.io/os: linux
initContainers:
  - image: velero/velero-plugin-for-microsoft-azure:v1.6.0
    imagePullPolicy: IfNotPresent
    name: velero-velero-plugin-for-microsoft-azure
    resources:
      limits:
        cpu: 200m
        memory: 128Mi
      requests:
        cpu: 50m
        memory: 32Mi
    volumeMounts:
      - mountPath: /target
        name: plugins
credentials:
  useSecret: true
  existingSecret: velero-credentials
configuration:
  provider: azure
  restoreOnlyMode: false
  defaultVolumesToFsBackup: true
  uploaderType: kopia
  backupStorageLocation:
    name: default
    bucket: velero
    config:
      subscriptionId: subid
      storageAccount: "storage_account_name"
      resourceGroup: "azure_resource_group_where_storage_account_is_created"

  volumeSnapshotLocation:
    name: default

schedules:
  default:
    schedule: "0 0 * * *"
    useOwnerReferencesInBackup: false
    template:
      ttl: "720h"
      includedNamespaces:
        - '*'
      excludedNamespaces:
        - "calico-system"
        - "default"
        - "kube-node-lease"
        - "kube-public"
        - "kube-system"
      excludedResources:
        - events
        - events.k8s.io
        - nodes
        - backups.velero.io
        - restores.velero.io
        - resticrepositories.velero.io
        - controllerrevision.apps
      snapshotVolumes: true
      storageLocation: default
      volumeSnapshotLocations:
        - default
```

I do not want to share full velero debug as sensitive information may be shared.

**Anything else you would like to add:**
Observed folders under kopia at Storage Account with names of namespaces which should not be backup.
Observed on grafana that node agents are run in sequence to backup persistent volumes.

**Environment:**

- Velero version (use `velero version`):  1.10.1, helm chart 3.1.2
- Velero features (use `velero client config get features`):  Azure plugin 1.6.0
- Kubernetes version (use `kubectl version`): AKS 1.24.6
- Kubernetes installer & version: AKS
- Cloud provider or hardware configuration: Azure
- OS (e.g. from `/etc/os-release`): Ubuntu 18 as defined by AKS

**Vote on this issue!**

This is an invitation to the Velero community to vote on issues, you can see the project's [top voted issues listed here](https://github.com/vmware-tanzu/velero/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc).  
Use the "reaction smiley face" up to the right of this comment to vote.

- :+1: for "I would like to see this bug fixed as soon as possible"
- :-1: for "There are more important bugs to focus on right now"

**Comment by allenxu404** [comment]
I tested in my local env and didn't hit this issue. Could you double-check it by simply running `velero backup describe <your backup> --details` then checking if there is any excludedNamespaces resources showing up in Resource List or checking it from S3 bucket?

**Comment by slawekww** [comment]
Yes, double checked and my excluded namespaces are not on the list - thanks for pointing it! It is my fault.

Have you got chance to look into Kopia schedule performance and observe schedule nodes backup one by one?
If it is intentionally, let close isse. 
If not, maybe it is option for improvement backup using Kopia.

**Comment by reasonerjt** [comment]
@slawekww 
I'm closing this issue as usage error, for the new requirement please open a new issue and we can triage properly.
